patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -443,6 +443,11 @@
 +
 +
 +    /* ======================================== FORGE START =====================================*/
```

Comment:
No, you shouldn't. Implementing hard interfaces on tiles instead of using tile capabilities isn't deprecated either.